tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -743,8 +744,66 @@ protected function getEntityManager(?Connection $connection = null, ?MappingDriv
 
         $config->setMetadataDriverImpl(
             $mappingDriver ?? $config->newDefaultAnnotationDriver([
+                realpath(__DIR__ . '/Models/CMS'),
```

Comment:
Wondering why this test changed to load everything - is this because we don't yet have a lazy collection?

Reply:
Just because I am pocking around a bit everywhere to try and fix everything that I broke.